test/specs/anInitialSuite.e2e.ts

```diff
@@ -25,25 +25,26 @@ suite does run, it needs to run first.
 describe('An Initial Suite', async () => {
   let testSetup: TestSetup;
 
+  step('Countdown', async () => {
+    utilities.log('About to start the e2e tests...');
+    for (let i = 10; i > 0; i--) {
+      utilities.log(`${i}...`);
+      await utilities.pause(1);
+    }
+  });
+
   step('Verify our extensions are not initially loaded', async () => {
     const workbench = await browser.getWorkbench();
-    await utilities.runCommandFromCommandPrompt(
-      workbench,
-      'Developer: Show Running Extensions',
-      2
-    );
+    await utilities.showRunningExtensions(workbench);
 
     const extensionNameDivs = await $$('div.name');
     let sfdxKeywordWasFound = false;
     let salesforceKeywordWasFound = false;
     for (const extensionNameDiv of extensionNameDivs) {
       const text = await extensionNameDiv.getText();
-
       if (text.includes('sfdx')) {
         sfdxKeywordWasFound = true;
-        utilities.log(
-          `AnInitialSuite - extension ${text} was present, but wasn't expected`
-        );
+        utilities.log(`AnInitialSuite - extension ${text} was present, but wasn't expected`);
```

Comment:
One of the reasons why I increased the wrap around length is that man y of the log statements were taking up multiple lines, and 80 chars seems too low.